### DISCUSSION TITLE
Is it Ad Free?????

### DISCUSSION THREAD

**Discussion by ThanosAtha** [pear-devs/pear-desktop#730] [General]
OMG. YouTube could have a Desktop app without ads, too!
Both of them on Android exist, (just for info).....

**Comment by iliqiliev** [comment]
Yes it is, the adblock plugin is enabled by default

**Comment by inson1** [comment]
@ThanosAtha What about closing this discussion? :) Its year and half :)

And yes, thank you th-ch for all your hard work :)
